docs/Readme.md

```diff
@@ -27,24 +29,31 @@
     [Heuristic](Learning-Environment-Design-Heuristic-Brains.md),
     [Learning](Learning-Environment-Design-Learning-Brains.md)
 * [Learning Environment Best Practices](Learning-Environment-Best-Practices.md)
-* [Using the Monitor](Feature-Monitor.md)
-* [Using the Video Recorder](https://github.com/Unity-Technologies/video-recorder)
-* [Using an Executable Environment](Learning-Environment-Executable.md)
-* [Creating Custom Protobuf Messages](Creating-Custom-Protobuf-Messages.md)
+
+Optional for first time users
+  * [Using the Monitor](Feature-Monitor.md)
+  * [Using the Video Recorder](https://github.com/Unity-Technologies/video-recorder)
+  * [Using an Executable Environment](Learning-Environment-Executable.md)
+  * [Creating Custom Protobuf Messages](Creating-Custom-Protobuf-Messages.md)
 
 ## Training
 
 * [Training ML-Agents](Training-ML-Agents.md)
+* [Using TensorBoard to Observe Training](Using-Tensorboard.md)
 * [Training with Proximal Policy Optimization](Training-PPO.md)
+
+### Advanced Training Methods
+
 * [Training with Soft Actor-Critic](Training-SAC.md)
 * [Training with Curriculum Learning](Training-Curriculum-Learning.md)
 * [Training with Imitation Learning](Training-Imitation-Learning.md)
 * [Training with LSTM](Feature-Memory.md)
 * [Training Generalized Reinforcement Learning Agents](Training-Generalized-Reinforcement-Learning-Agents.md)
+* [Training Using Concurrent Unity Instances](Training-Using-Concurrent-Unity-Instances.md)
+
+### Cloud Training 
```

Comment:
@xiaomaogy - have we tested the AWS/Azure pages? I'd like us to message those pages more carefully. As a use-at-your-own-risk.

Reply:
@mmattar  Do you want to explicitly state "Use at your own risk" in the Cloud Training section or was that a figure of speech? We can also say it's Under Development or Beta?

Reply:
I think either works, but some indication that we cannot guarantee performance.

Reply:
If we aren't testing these cloud set-ups ourselves, we should be cautious about seeming to recommend them. I would change the heading to "### Cloud Training (Unsupported)" and then add a line that says something like: "Here are cloud training set-up guides for Azure and AWS. We no longer use them ourselves and so they may not be work correctly. We've decided to keep them up just in case they are helpful to you. If you do try them out, please share your experience with us, and feel free to submit a PR with any necessary changes." This then leaves the room for us to provide a supported GCP set-up. @unityjeffrey - thoughts?

Reply:
@mmattar We don't test the AWS/Azure pages for every release. AWS page was tested during the 0.7 release, and we've never tested Azure page (and it was contributed by an external PR from someone in Microsoft). 

I think use-at-your-own-risk makes more sense than "under development", because we are not actively working on it nor making sure it is working for every release.